imagebuildah/build.go

```diff
@@ -380,6 +380,7 @@ func (b *Executor) Run(run imagebuilder.Run, config docker.Config) error {
 		Entrypoint:      config.Entrypoint,
 		Cmd:             config.Cmd,
 		NetworkDisabled: config.NetworkDisabled,
+		QuietBuild:      b.quiet,
```

Comment:
Why not just call this Quiet?  Change the value to Quiet and then it LGTM

Reply:
Changed to Quiet and just pushed.  Tests are cranking.